utils/service_discovery/abstract_config_store.py

```diff
@@ -125,17 +125,17 @@ def read_config_from_store(self, identifier):
             except KeyNotFound:
                 log.debug("Could not find directory {} in the config store, "
                           "trying to convert to the old format.".format(identifier))
-                legacy_ident = identifier.split(':')[0].split('/')[-1]
-                res = self._issue_read(legacy_ident)
+                image_ident = self._get_image_ident(identifier)
```

Comment:
In this case it's used for backward compatibility so I didn't change the debug message. Same thing 10 lines below. We shouldn't rely on this as the main identifier as it's not fine grained enough.